bokeh/command/subcommands/serve.py

```diff
@@ -198,7 +198,7 @@ def invoke(self, args):
 
         server_kwargs = { key: getattr(args, key) for key in ['port',
                                                               'address',
-                                                              'keep_alive']
+                                                              'keep_alive_milliseconds']
                           if getattr(args, key, None) is not None }
```

Comment:
I think you can configure argparse to use a different name on `args` but lets find a way to clean all this up later